src/main/java/core/basesyntax/serviceimpl/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReaderService {
+    @Override
+    public List<String> read(String fileName) {
+        try {
+            return Files.readAllLines(Path.of(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file : " + fileName, e);
+        }
+    }
```

Comment:
Make sure to place the input files into the 'src/main/resources' folder. When providing the file path, use relative paths instead of absolute ones to ensure that the project runs on all environments.